tt-train/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.18)
+cmake_policy(VERSION 3.18)
+include(cmake/compilers.cmake)
```

Comment:
The root of the repo should be setting the defaults for compiler options.
```suggestion
```

Reply:
I'm not sure what to do here.
If we remove this, tt-train could probably not build independently as it used to.
I don't really care about that though.
@dmakoviichuk-tt do you?

Reply:
We should be able to build it independently.  But I am ok to reuse compilers cmake from the tt-metal.